### PR TITLE
CC for LPT transformation call in CPU plugin

### PR DESCRIPTION
**Pull request by egorshul**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *71539*

### CODE REVIEW

File: src/plugins/intel_cpu/src/plugin.cpp [hunk 1]
```diff
@@ -432,6 +433,7 @@ static void TransformationUpToCPUSpecificOpSet(std::shared_ptr<ngraph::Function>
 
     using namespace ngraph::pass::low_precision;
     if (useLpt) {
+        CPU_LPT_SCOPE(LowPrecisionTransformations);
```
Comment (ilya-lavrenov) [inline src/plugins/intel_cpu/src/plugin.cpp:436]: we have several `if (useLpt)` in this file, should we use this `CPU_LPT_SCOPE` in all such places?

Comment (egorshul) [reply src/plugins/intel_cpu/src/plugin.cpp]: Fixed in cbaf98dc955a0dcbf9a2bc649a271fab97eb192e. Thank you!

**Comment by ilyachur** [comment]
Did you check that CC works? Do you have any numbers?

**Comment by egorshul** [comment]
@ilyachur If we build OpenVINO as a dynamic libraries using conditional compilation, we get:
[AlexNet FP32] libopenvino.so: 10,7 Mb -> 10,1 Mb
[AlexNet INT8] libopenvino.so: 10,8 Mb -> 10,4 Mb

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**